com/RF_SHUT.csh

```diff
@@ -1,14 +1,17 @@
 #!/usr/local/bin/tcsh -f
 #################################################################
+#                                                               #
+# Copyright (c) 2002-2016 Fidelity National Information         #
+# Services, Inc. and/or its subsidiaries. All rights reserved.  #
 #								#
-# Copyright (c) 2018 YottaDB LLC. and/or its subsidiaries.	#
+# Copyright (c) 2017-2018 YottaDB LLC. and/or its subsidiaries.	#
 # All rights reserved.						#
-#								#
-#	This source code contains the intellectual property	#
-#	of its copyright holder(s), and is made available	#
-#	under a license.  If you do not know the terms of	#
-#	the license, please stop and do not read further.	#
-#								#
+#                                                               #
```

Comment:
Tabs all through the copyright notice have been replaced with spaces. Please undo that.